install/third-party/flutter/install.yml

```diff
@@ -0,0 +1,14 @@
+id: flutter-mobile
+name: Flutter
+title: Flutter Monitoring
+description: |
+  New Relic's mobile monitoring for Flutter gives you a comprehensive view over all of your app's performance. It works for apps using Version >=2.0
+
+target:
+  type: agent
+  destination: application
+
+install:
+  mode: link
+  destination:
+    url: https://github.com/jake-landersweb/flutter-nri-agent/blob/434a91cc8291f278b743f98a71e82da838f5edde/install.md
```

Comment:
Since these instructions are on a personal github account, let's either get them moved to an official new relic repo or possibly update the quickstart description to include those details. Here we could just link out to the docs on installing flutter monitoring.